docs/source/advanced/js-interop.mdx

```diff
@@ -0,0 +1,234 @@
+---
+title: JS Interoperability
+---
```

Comment:
I reshuffled this file a bit, I hope you don't mind.

Reply:
It looks great!